finetune/target_models/masked_language_model.py

```diff
@@ -96,6 +99,111 @@ def __init__(self, *args, **kwargs):
     def _get_input_pipeline(self):
         return MaskedLanguageModelPipeline(self.config)
 
+
+    def display_text_(self, input_text, k=5, context=None, **kwargs):
+
+        def dataset_encoded():
+            while not dataset_encoded.finished:
+                yield {"tokens": arr_encoded.token_ids, "mask": arr_encoded.mask}
+
+        dataset_encoded.finished = False
+
+        def get_input_fn():
+            types, shapes = self.input_pipeline.feed_shape_type_def()
+            tf_dataset = Dataset.from_generator(dataset_encoded, types[0], shapes[0])
+            return tf_dataset.batch(1)
+
+        encoded = self.input_pipeline.text_encoder._encode([input_text])
+        token_ids = encoded.token_ids[0]
+        encoded = EncodedOutput(token_ids=token_ids)
+        estimator, hooks = self.get_estimator(force_build_lm=True)
+        predict = estimator.predict(input_fn=get_input_fn,
+                                    predict_keys=[PredictMode.GENERATE_TEXT,],
+                                                 # PredictMode.MLM_IDS,
+                                                 # PredictMode.MLM_POSITIONS],
+                                                  hooks=hooks)
+        arr_encoded = self.input_pipeline._array_format(encoded)
+        prediction_information = list(predict)
+
+        return prediction_information
+
+
+    def display_text(self, input_text, k=5, context=None, **kwargs):
```

Comment:
Feel like we can probably come up with a better name than `display_text` that's more descriptive of what's happening here... but I'm not sure what it is.

Reply:
going with predict_top_k_report, unless you think predict_topk_report is better?